content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -1,72 +1,240 @@
+for each product, including which package each item is shipped out in.
 
-The packaging is the physical container that protects your product. If
-you are selling computers, the packaging contains the computer with the
-notice and the power plug.
+To use this method, ensure the :guilabel:`Packages` option is enabled, by navigating to
+:menuselection:`Inventory app --> Configuration --> Settings --> Operations`. Click the
+:guilabel:`Packages` check box, and then :guilabel:`Save` to activate the feature.
 
-In Odoo, packagings are used for indicative purposes on sale orders.
-They can be specified on the product form, in the inventory tab.
+On a :abbr:`(Delivery Order)`, assign which package to use by clicking on the :guilabel:`≣ (Detailed
+Operations)` icon to the right of each product in the :guilabel:`Operations` tab.
 
-.. image:: usage/usage_07.png
-    :align: center
+.. image:: usage/detailed-operations-icon.png
+   :align: center
+   :alt: Find detailed operations icon to the right on the product line.
 
-.. image:: usage/usage_08.png
-    :align: center
+Clicking the :guilabel:`≣ (Detailed Operations)` icon of a product reveals a pop-up window. In this
+window, the :guilabel:`Product` name, :guilabel:`Demand`, and :guilabel:`Quantity Done` are clearly
+displayed above the customizable product line.
 
-.. note::
-        Another useful use of the packaging is for product reception. By
-        scanning the barcode of the packaging, Odoo adds the number of units
-        contained in the packing on the picking.
+Here, the warehouse location the product is coming from can be modified, in addition to the
+:guilabel:`Source Package` and :guilabel:`Destination Package`, which represents what package(s) the
+product(s) will be packaged into.
+
+Click :guilabel:`Add a line` to include each additional package used. Enter the desired amount in
+the :guilabel:`Done` field to specify the amount of products go into the :guilabel:`Destination
+Package`.
+
+Multiple packages in a single delivery
+--------------------------------------
+
+Although a :abbr:`DO (Delivery Order)` is typically associated with a single package, orders can be
+split by clicking the :guilabel:`≣ (Detailed Operations)` icon next to the products on a :abbr:`SO
+(Sales Order)` line. This opens the :guilabel:`Detailed Operations` window with an editable table to
+specify which products are intended for which package.
+
+.. example::
+   To package 10 boxes of pencils with 2 boxes of erasers from the same :abbr:`SO (Sales Order)`,
+   select the :guilabel:`≣ (Detailed Operations)` icon to the right of `Box of Pencils`. This opens
+   a window that is used to detail how products, like the 20 `Box of Pencils`, are packaged.
+
+   Fill in `10` under the :guilabel:`Done` field to package 10 products into the
+   :guilabel:`Destination Package`. Start typing under this field and select the :guilabel:`Create`
```

Comment:
"this" field.
which field?
is this a new order line?

Reply:
Btw I understand how this works since I'm familiar with Odoo, however, for new users, it might be worth spelling this example out more by removing "this/that" language and specifying where we are, what's happening, or what's about to happen (guis can help here, e.g. name of pop-up window, column name for a field, etc).

Reply:
Rewrote to
`Type in `10` under the :guilabel:`Done` field to package 10 products into the
   :guilabel:`Destination Package`. To specify the destination package, type the name of an existing
   package (such as `PACK0000006`) under the field, :guilabel:`Destination Package`. If the package
   does not exist, Odoo displays the :guilabel:`Create` button to create a new package. Click
   :guilabel:`Add a line` to assign the remaining products in another package, `PACK0000007`.
`